.github/workflows/lint.yml

```diff
@@ -15,18 +15,11 @@ jobs:
         with:
           go-version: 'stable'
           cache: false
-      - name: go mod pakcage cache
-        uses: actions/cache@v4
-        with:
-          path: ~/go/pkg/mod
-          key: ${{ runner.os }}-go-${{ hashFiles('go.mod') }}
```

Comment:
Why remove this?

Reply:
> Why remove this?

Seems not need, no difference in speed

Reply:
Why no difference? Is it the wrong way to use it? We hope it can improve the script speed.

Reply:
> Why no difference? Is it the wrong way to use it? We hope it can improve the script speed.

I will check it.